lib/iris/experimental/ugrid/mesh.py

```diff
@@ -122,25 +122,31 @@ def __init__(
         units=None,
         attributes=None,
         start_index=0,
-        src_dim=0,
+        location_axis=0,
     ):
         """
         Constructs a single connectivity.
 
         Args:
 
         * indices (numpy.ndarray or numpy.ma.core.MaskedArray or dask.array.Array):
-            The index values describing a topological relationship. Constructed
-            of 2 dimensions - the list of locations, and within each location:
-            the indices of the 'target locations' it relates to.
-            Use a :class:`numpy.ma.core.MaskedArray` if :attr:`src_location`
-            lengths vary - mask unused index 'slots' within each
-            :attr:`src_location`. Use a :class:`dask.array.Array` to keep
-            indices 'lazy'.
+            2D array giving the topological connection relationship between
+            ' :attr:`location` elements ' and ' :attr:`connected` elements '.
```

Comment:
The extra "ordinary" quotes here are not wanted + look odd when I build it.

Reply:
7471929b585df6ff8718a75b2224b1027a5331d2